Objective-J/CommonJS/lib/objective-j/compiler.js

```diff
@@ -259,12 +259,16 @@ exports.main = function(args)
         if (argv[0] === "--help" || argv[0].substr(0, 1) == '-')
         {
             print("Usage (objjc 2.0): " + args[0] + " [options] [--] file...");
-            print("  -p, --print                    print the output directly to stdout");
-            print("  --unmarked                     don't tag the output with @STATIC header");
+            print("  -p, --print                         print the output directly to stdout");
+            print("  --unmarked                          don't tag the output with @STATIC header");
             print("");
-            print("  -T, --includeTypeSignatures    include type signatures in the compiled output");
+            print("  -T, --dont-include-type-signatures  include type signatures in the compiled output");
```

Comment:
The use case is if you want to hide how your application works. Usually you minify/obfuscate the Javascript code, remove debug symbols and remove the type information. Sometimes your application need the type information and you have to leave it there.